### ISSUE TITLE
Activate snapshots and review packaging

### ISSUE THREAD

**Issue by javiertuya** [giis-uniovi/modevo#26]
Start after #21 is completed

**Comment by giis-qabot** [comment]
@javiertuya This is a **reminder** about this issue because it has not been updated for **10 days**

**Comment by javiertuya** [comment]
I was working in this issue but I realized that the full lifecycle of the project has errors at the install phase of modevo-transform, therefore, I will close it.
@suarezgpablo If at some point this is solved, please, reopen the issue.

```
mvn clean install -DskipTests=true
...
[INFO] Reactor Summary for modevo 1.0.1-SNAPSHOT:
[INFO]
[INFO] modevo ............................................. SUCCESS [  0.462 s]
[INFO] modevo-transform ................................... SUCCESS [ 25.674 s]
[INFO] modevo-script ...................................... FAILURE [  0.415 s]
[INFO] modevo-bom ......................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  26.759 s
[INFO] Finished at: 2024-05-07T19:42:02+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project modevo-script: Could not resolve dependencies for project io.github.giis-uniovi:modevo-script:jar:1.0.1-SNAPSHOT: The following artifacts could not be resolved: io.github.giis-uniovi:modevo-transform:jar:tests:1.0.1-SNAPSHOT (absent): Could not find artifact io.github.giis-uniovi:modevo-transform:jar:tests:1.0.1-SNAPSHOT -> [Help 1]
```

**Comment by javiertuya** [comment]
Labeled as wontfix
